lib/diagnostic.js

```diff
@@ -17,7 +17,16 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+/*
+Entity: Calender
+Attributes: day, month, year, time
+Methods: onScheduleToday(user) = Whats on your schedule today
+
+Entity: User
+Attributes: name, schedule
```

Comment:
What is the difference between a schedule and a calendar in this set of abstractions?